ucdlib-directory/includes/people.php

```diff
@@ -283,7 +300,180 @@ public function register_post_meta(){
       'default' => false,
       'type' => 'boolean',
     ) );
+  }
+  
+  // ensure block exists
+  public function ensureBlockExists($block_name, $dry_run=false, $inCLI=false, $insert_after_block='') {
+    $post_type = $this->slug; 
+    if($inCLI) echo "\n== Directory People: Ensure {$block_name} block ==";
+
+    if (empty($block_name)) {
+      if($inCLI) echo "\n Block Name is empty, please give a valid blockname'.";
+      else {
+          $blockNameMessage = wp_json_encode("Block Name is empty, please give a valid blockname");   
+          echo "<script>console.warn({$blockNameMessage});</script>";
```

Comment:
if not run from the cli, the method should be SILENT. echoing anything will interfere with wordpress - only very specific hooks can write to the DOM e.g. https://developer.wordpress.org/reference/hooks/wp_head/